03 Algorithm Framework/02 Universe Selection/02 Manual Universe Selection.html

```diff
@@ -1,7 +1,7 @@
 <p>
 Manual universe selection chooses a static, fixed set of assets to trade. This is most useful when selecting a set of currencies, or a basket of ETF stocks. You can also add assets with the traditional <a href="/docs/algorithm-reference/initializing-algorithms#Initializing-Algorithms-Selecting-Asset-Data" class="docs-internal-link">AddSecurity API</a>.
 </p>
-<p>The <code>ManualUniverseSelectionModel</code> class is initialized with an array of <code>Symbol</code> objects at the UniverseSettings.Resolution data resolution. $[Symbol] objects can be created with the Symbol.Create method.
```

Comment:
`$[ManualUniverseSelectionModel, T:QuantConnect.Algorithm.Framework.Selection.ManualUniverseSelectionModel]`.
Both Symbol should have a link. `<code>Symbol</code> -> $[Symbol,T:QuantConnect.Symbol]`,

Reply:
https://www.quantconnect.com/services/inspector?type=T:QuantConnect.Algorithm.Framework.Selection.ManualUniverseSelectionModel
I can't find the framework method in class doc

Reply:
Probably the inspector is not up to date and does not include these new models yet